openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tagging API Views
+"""
+from rest_framework.viewsets import ModelViewSet
+
+from openedx.core.lib.api.view_utils import DeveloperErrorViewMixin
```

Comment:
By design, we can't depend on code in edx-platform in this repo -- it needs to stand alone. cf https://github.com/openedx/openedx-learning/blob/main/docs/decisions/0001-purpose-of-this-repo.rst